ecl/eclcc/eclcc.cpp

```diff
@@ -1078,13 +1081,18 @@ static bool findFilenameInSearchPath(StringBuffer & attributePath, const char *
                     tail++;
                 if (convertPathToModule(attributePath, tail))
                     return true;
+                else
```

Comment:
Bug unrelated to this issue: Line 1080-1081 should really be:

```
if (isPathSepChar(*tail))
    tail++
else if (*tail!=nullptr) 
    break; // to avoid matching part of the filename in expandedSourceName
```